biorbd_optim/__init__.py

```diff
@@ -8,4 +8,6 @@
 from .dynamics import Dynamics
 from .mapping import Mapping, BidirectionalMapping
 from .optimal_control_program import OptimalControlProgram
+from .biorbd_interface import BiorbdInterface
+from .tests import Tests
```

Comment:
This should not be accessible for people either. 
Moreover I don't understand the purpose of this class. It should only be a simple file in the tests folder, not something related to the API itself

Reply:
There is no need to encapsulate them in a class